cmd/notation/sign.go

```diff
@@ -50,6 +59,12 @@ Example - Sign an OCI artifact identified by a tag (Notation will resolve tag to
 
 Example - Sign an OCI artifact stored in a registry and specify the signature expiry duration, for example 24 hours
   notation sign --expiry 24h <registry>/<repository>@<digest>
+
+Example - Sign an OCI artifact and use OCI image manifest to store the signature, with the default JWS envelope:
+  notation sign --signature-manifest image <registry>/<repository>@<digest>
+
+Example - Sign an OCI artifact and use OCI image manifest to store the signature, with the COSE envelope:
+  notation sign --signature-manifest image --signature-format cose <registry>/<repository>@<digest> 
```

Comment:
IMO we dont need to show examples for all the signature envelope

```suggestion
Example - Sign an OCI artifact and use OCI image manifest to store the signature:
  notation sign --signature-manifest image <registry>/<repository>@<digest>
```